### ISSUE TITLE
unused variables - false positive when usage is inside MESSAGE statement

### ISSUE THREAD

**Issue by FreHu** [abaplint/abaplint#2411]
```abap
REPORT zfoobar.

CLASS lcx_syntax_error DEFINITION
  CREATE PRIVATE.

  PUBLIC SECTION.

    CLASS-METHODS invalid_calculation_attempt
      IMPORTING operator TYPE string
                left     TYPE string
                right    TYPE string
                previous TYPE REF TO cx_root.
  PRIVATE SECTION.


ENDCLASS.


CLASS lcx_syntax_error IMPLEMENTATION.

  METHOD invalid_calculation_attempt.
    RAISE EXCEPTION TYPE lcx_syntax_error MESSAGE e016 WITH operator left right
      EXPORTING previous = previous.
  ENDMETHOD.

ENDCLASS.
```

operator, left right are reported as unused variables

**Comment by larshp** [comment]
thanks 👍

**Comment by larshp** [comment]
with less whitespace,

```abap
CLASS lcx_syntax_error DEFINITION CREATE PRIVATE.
  PUBLIC SECTION.
    CLASS-METHODS invalid_calculation_attempt
      IMPORTING operator TYPE string
                left     TYPE string
                right    TYPE string
                previous TYPE REF TO cx_root.
ENDCLASS.

CLASS lcx_syntax_error IMPLEMENTATION.
  METHOD invalid_calculation_attempt.
    RAISE EXCEPTION TYPE lcx_syntax_error MESSAGE e016 WITH operator left right
      EXPORTING previous = previous.
  ENDMETHOD.
ENDCLASS.
```

**Comment by larshp** [comment]
@FreHu try again, 2.85.19
